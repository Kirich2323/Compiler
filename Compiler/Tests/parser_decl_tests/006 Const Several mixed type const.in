const
    a = 0;
    b = 10.5;
    c = -2;
    ConstName = 24e-2;