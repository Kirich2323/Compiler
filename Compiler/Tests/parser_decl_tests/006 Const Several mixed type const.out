main
----------
char                     type 
float                    type 
integer                  type 
a               const integer               0
b                 const float            10.5
c               const integer              -2
ConstName         const float            0.24